src/guide/essentials/template-refs.md

```diff
@@ -314,6 +314,8 @@ defineExpose({
 
 When a parent gets an instance of this component via template refs, the retrieved instance will be of the shape `{ a: number, b: number }` (refs are automatically unwrapped just like on normal instances).
 
+Note that defineExpose must be called before any await operation. Otherwise, properties and methods exposed after the await operation will not be accessible. ([Related Discussion](https://github.com/vuejs/core/issues/4930))
```

Comment:
```suggestion
Note that defineExpose must be called before any await operation. Otherwise, properties and methods exposed after the await operation will not be accessible. 
```